### PR TITLE
List system containers

### PR DESCRIPTION
**Pull request by tanyasethi-msft**
- [x] The purpose of this PR is explained in this or a referenced issue.
- [x] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [x] Tests are included and/or updated for code changes.
- [x] Updates to module CHANGELOG.md are included.
- [x] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go

The functionality of listing System File systems already exist, adding appropriate tests as part of this PR

### CODE REVIEW

File: sdk/storage/azdatalake/service/client_test.go [hunk 1]
```diff
@@ -695,6 +695,26 @@ func (s *ServiceRecordedTestsSuite) TestListFilesystemsBasicUsingConnectionStrin
...
 
+func (s *ServiceRecordedTestsSuite) TestListFilesystemsIncludeSystemFileSystems() {
+	_require := require.New(s.T())
+	svcClient, err := testcommon.GetServiceClient(s.T(), testcommon.TestAccountDatalake, nil)
+	_require.NoError(err)
+
+	listOptions := service.ListFileSystemsOptions{Include: service.ListFileSystemsInclude{System: to.Ptr(true)}}
+	count := 0
+	pager := svcClient.NewListFileSystemsPager(&listOptions)
+	for pager.More() {
```
Comment (vibhansa-msft) [inline sdk/storage/azdatalake/service/client_test.go:706]: Was this only a miss in test ?

**Review by souravgupta-msft — APPROVED** [review]

File: sdk/storage/azdatalake/service/client_test.go (hunk 1 above)
Comment (tanyasethi-msft) [reply sdk/storage/azdatalake/service/client_test.go:706]: Yes

**Pull request merged**